### PR TITLE
fix: Populate null values

### LINKED ISSUE #2451: Error from GoEthClient when calling getBlockByNumber on certain specific blocks
### Description

GoEthClient returns error on certain specific block for some reason. 

Here are some of the errors
```
2024/05/06 10:47:32 json: cannot unmarshal non-string into Go struct field rpcBlock.transactions of type hexutil.Uint64
exit status 1
```

```
2024/05/06 10:47:55 missing required field 'maxPriorityFeePerGas' for txdata
exit status 1
```

However, some blocks are working fine. 

### Steps to reproduce

You can try to run this repo to reproduce the problem.

https://github.com/pathornteng/hedera-ethclient-problem

### Additional context

_No response_

### Hedera network

testnet

### Version

latest

### Operating system

None

### PR DESCRIPTION
**Pull request by ebadiere**
Populate null values.

**Related issue(s)**:

Fixes #2451

### CODE REVIEW

File: packages/relay/package.json [hunk 1]
```diff
@@ -56,6 +56,7 @@
     "better-lookup": "^1.3.0",
     "buffer": "^6.0.3",
     "dotenv": "^16.0.0",
+    "ethereum-utils": "^1.0.0",
```
Comment (AlfredoG87) [inline packages/relay/package.json:59]: why we need this dep?

File: packages/relay/src/formatters.ts [hunk 2]
```diff
@@ -23,6 +23,7 @@ import crypto from 'crypto';
 import { Transaction, Transaction1559, Transaction2930 } from './lib/model';
 import { BigNumber } from '@hashgraph/sdk/lib/Transfer';
 import { BigNumber as BN } from 'bignumber.js';
+import { addHexPrefix, BN as BigNum, intToHex } from 'ethereumjs-util';
```
Comment (AlfredoG87) [inline packages/relay/src/formatters.ts:26]: what is wrong w current BN?

**Review by AlfredoG87 — APPROVED** [review]

**Review by AlfredoG87 — COMMENTED** [review]
seems like some tests are failing

**Comment by ebadiere** [comment]
Replaced by: #2499

**Pull request closed without merging**